### PR TITLE
Update tp-app-payment-gateway-dev.yaml

### PR DESCRIPTION
**Pull request by shashwattechsophy**

### CODE REVIEW

File: tp-app-payment-gateway-dev.yaml [hunk 1]
```diff
@@ -23,6 +23,15 @@ paymentgateway:
     keysecret: ${PAYPAL_KEY_SECRET:EMuFpv1mkjSavIVveEdq3zxLMjuDl8SOPYLjl8dgtRhDqI2zXK-YT3l95jpPbk-uS1mFmyM6r1RnTvc5}
```
Comment (VaibhavTechsophy) [inline tp-app-payment-gateway-dev.yaml:23]: donot commit secrets, please remove

File: tp-app-payment-gateway-dev.yaml [hunk 2]
```diff
@@ -23,6 +23,15 @@ paymentgateway:
...
     bncode: ${PAYPAL_BN_CODE:Creed_SP_PPCP}
   payby:
+  
+  paytm:
+    url: ${PAYTM_URL:https://securegw-stage.paytm.in}
+    callback-url: ${PAYTM_CALLBACK_URL:http:localhost/8080}
+    merchantId: ${PAYTM_MERCHANT_ID:XXOqwZ74462439658697}
+    merchantKey: ${PAYTM_MERCHANT_KEY:GoQzDnAqx4#mQugj}
```
Comment (VaibhavTechsophy) [inline tp-app-payment-gateway-dev.yaml:31]: no merchant key should be configured here

Comment (VaibhavTechsophy) [reply tp-app-payment-gateway-dev.yaml:31]: same for merchant id, it should only be present in environment

File: tp-app-payment-gateway-dev.yaml (hunk 1 above)
Comment (shashwattechsophy) [reply tp-app-payment-gateway-dev.yaml:23]: this was already there and i think is being used

File: tp-app-payment-gateway-dev.yaml (hunk 2 above)
Comment (shashwattechsophy) [reply tp-app-payment-gateway-dev.yaml]: have made changes

**Review by VaibhavTechsophy — APPROVED** [review]

**Pull request merged**